public/locales/en-US.json

```diff
@@ -29,9 +29,9 @@
     "cardHeading_Satellite Capsule": "CPU socket usage",
     "cardHeading_Satellite Server": "CPU socket usage",
     "cardHeadingDescription": "...",
-    "cardHeadingDescription_Instance-hours": "Instance hours usage in gibibyte hours",
-    "cardHeadingDescription_Storage-gibibytes": "Data storage usage in gibibyte hours",
-    "cardHeadingDescription_Transfer-gibibytes": "Gibibyte hour usage per instance",
+    "cardHeadingDescription_Instance-hours": "Instance hours usage in binary gigabyte hours",
+    "cardHeadingDescription_Storage-gibibytes": "Data storage usage in binary gigabyte hours",
+    "cardHeadingDescription_Transfer-gibibytes": "Binary Gigabyte hour usage per instance",
```

Comment:
@jlprevatt confirm this changes towards
```
"cardHeadingDescription_Transfer-gibibytes": "Binary gigabyte hour usage per instance"
```

Reply:
Data transfer is NOT counted over time, just counted (the old "always going up" thing), so:
"Data transfer usage in binary gigabytes"